scripts/ci/build-and-test.sh

```diff
@@ -10,6 +10,26 @@ cd $(dirname $0)/../..
 source scripts/ci/sources/mode.sh
 source scripts/ci/sources/tunnel.sh
 
+# Get commit diff
+if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then
+  diff=$(git diff --name-only $TRAVIS_COMMIT_RANGE)
+else
+  diff=$(git diff --name-only $TRAVIS_BRANCH...HEAD)
+fi
+
+# Check if tests can be skipped
+skip_tests=true
+for filename in $diff; do
+  if ! [[ "$filename" =~ .*\.md ]]; then
+    skip_tests=false
+    break
+  fi
+done
```

Comment:
Thinking if we could make it more simple

```bash
if [[ "$TRAVIS_PULL_REQUEST" = "false" ]]; then
  fileDiff=$(git diff --name-only $TRAVIS_COMMIT_RANGE)
else
  fileDiff=$(git diff --name-only $TRAVIS_BRANCH...HEAD)
 fi

if [[ ${fileDiff} =~ ^(.*\.md\s*)*$ ]] && (is_e2e || is_unit); then
  echo "Skipping e2e and unit tests since only markdown files changed"
  exit 0
fi
```